### ISSUE TITLE
[FEAT]: Admin Dashboard - Replace aria-label CSS tooltip

### ISSUE THREAD

**Issue by gigamaster** [xoopscube/legacy#188]
### Description

On hover, some elements are accessible but not visible due to aria-label tooltip 
- move the tooltip to the top
- remove tooltip from elements with text
